docs/conf.py

```diff
@@ -19,6 +19,7 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../')))
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), 'rtd_hacks')))
```

Comment:
Oh sorry, this was there from when I was wrestling with trying to figure out how to get RTD to work. I think I'm going to give up on RTD now that we have the docs hosted at https://bodhi.stg.fedoraproject.org/docs/